openstack/baremetal/v1/nodes/results.go

```diff
@@ -50,6 +50,9 @@ func ExtractNodesInto(r pagination.Page, v interface{}) error {
 
 // Node represents a node in the OpenStack Bare Metal API.
 type Node struct {
+	// Whether automated cleaning is enabled or disabled on this node.
+	AutomatedClean *bool `json:"automated_clean"`
```

Comment:
Similar here, can you add a comment to note which microversion this field was added in? As an example:

https://github.com/gophercloud/gophercloud/blob/ed4deec00ff1d4d4c8a762af0c6360d4184a4bf4/openstack/compute/v2/servers/results.go#L221-L223

Reply:
Fixed, thank you.